web/src/beta/components/RadioGroup/index.tsx

```diff
@@ -4,7 +4,7 @@ import RadioBox from "@reearth/beta/components/RadioGroup/RadioBox";
 import { styled } from "@reearth/services/theme";
 
 export type Option = {
```

Comment:
Can we remove selected from Option and pass it as a single string prop whose value is the selected option's value?
If we do that, we can avoid the `isAtLeastOneSelected` stuff. 
We can then also replace currentOptions with currently selected state that does `useState(selected ?? options[0].value)`.